packages/types/src/api.ts

```diff
@@ -9,6 +9,7 @@ export interface ClerkAPIError {
     paramName?: string;
     sessionId?: string;
     emailAddresses?: string[];
+    identifiers?: string[];
```

Comment:
@chanioxaris Did we introduced that to FAPI ? I don't think we support any other identifier than emails, so is this even useful ?

Reply:
Allowlist & Blocklist are working for every identifier type (email_address, phone_number, web3_wallet). So in such an api error we can't limit them to only email addresses or reuse the existing meta parameter `emailAddresses`

Reply:
@chanioxaris Shall we update the copy here ? (Happy to do it as part of another PR)
![image](https://github.com/clerkinc/javascript/assets/19269911/34184ebc-2c2d-4ebd-9b3f-421c2c453f19)

Reply:
In the case of Organization Invitations, only email addresses can be provided. Although the API error is used in general for the Allowlist/Blocklist feature